aws/localstack/Spring-boot-example/customerregistration/pom.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please move the spring boot app directly into the folder "aws/localstack" (i.e. the pom.xml would be in aws/localstack.pom.xml).

Remove the cloudformation folder and instead put the cloudformation template next to the pom.xml.

Please update the links in the article accordingly.